src/widgets/NumericTextCtrl.cpp

```diff
@@ -235,6 +235,16 @@ void NumericTextCtrl::UpdateAutoFocus()
    }
 }
 
+bool NumericTextCtrl::SetTypeAndFormatName(const NumericConverterType& type, const NumericFormatSymbol& formatName)
+{
```

Comment:
This function shadows the base class function but does not override it.  Yet it does override other functions.

As you have mentioned, and I agree, ideally the control has-a converter, not is-a.  But, that's for future.